package.json

```diff
@@ -150,6 +150,7 @@
         "visible": true,
         "subPermissions": [
           "module.orders.enabled",
+          "addresstypes.collection.get",
```

Comment:
user with "settings" perms only will get an exception on the lack of this perm as well